ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -155,6 +156,24 @@ export const isBypassed = (
 ): boolean =>
   !!getAccessModuleStatusByName(profile, moduleName)?.bypassEpochMillis;
 
+export const displayModuleCompletionTime = (
```

Comment:
I wonder, is it better to replace `Time` string with `EpochMilli` in function name?

Reply:
Actually now that i am thinking about it, i feel it should be displayModuleCompletionDate since thats what is consistent with what we are showing on UI.